qms-tests/sanity/tests/documents/REQ-10.spec.ts

```diff
@@ -75,4 +75,53 @@ test.describe('ISO 13485, 4.2.4 Control of documents ensure that documents of ex
       await attachScreenshot('TESTS-347_manager_document_created.png', page)
     })
   })
+
+  test('TESTS-402. As a non space member, I cannot see nor edit any doc from that space', async ({ page }) => {
+    await allure.description(
+      'Requirement\nUser is not a part of space members and cannot see or edit any document from that space'
+    )
+    await allure.tms('TESTS-402', 'https://tracex.hc.engineering/workbench/platform/tracker/TESTS-402')
+    await test.step('2. check if non member can see space', async () => {
+      const folderName = faker.word.words(1)
+      const documentContentPage = new DocumentContentPage(page)
+      await documentContentPage.clickAddFolderButton()
+      await documentContentPage.fillDocumentSpaceFormManager(folderName)
+      await documentContentPage.changeDocumentSpaceMembers(folderName)
+      await documentContentPage.checkIfTheSpaceIsVisible(folderName, false)
+      await attachScreenshot('TESTS-402_space_not_existing.png', page)
+    })
+  })
+
+  test('TESTS-403. As a space member only, I cannot edit any doc from that space', async ({ page }) => {
+    await allure.description(
+      'Requirement\nUser is only part as a member and cannot see or edit any document from that space'
+    )
+    await allure.tms('TESTS-403', 'https://tracex.hc.engineering/workbench/platform/tracker/TESTS-403')
+    await test.step('2. check if non member edit or create a new doc in space', async () => {
+      const folderName = faker.word.words(1)
+      const documentContentPage = new DocumentContentPage(page)
+      const completeDocument: NewDocument = {
+        template: 'HR (HR)',
+        title: `Complete document-${generateId()}`,
+        description: `Complete document description-${generateId()}`
+      }
+      await documentContentPage.clickAddFolderButton()
+      await documentContentPage.fillDocumentSpaceFormManager(folderName)
+      await prepareDocumentStep(page, completeDocument, 1, undefined, folderName)
+      await documentContentPage.checkTeamMembersReviewNotExists()
+      await attachScreenshot('TESTS-403_member_cant_edit_space.png', page)
+    })
+  })
+
+  test('TESTS-404. As a space member only, I cannot create any doc from that space', async ({ page }) => {
+    await allure.description('Requirement\nUser is not able to create any document from that space')
+    await allure.tms('TESTS-404', 'https://tracex.hc.engineering/workbench/platform/tracker/TESTS-404')
+    await test.step('2. cCheck if user can not create documents as a space member', async () => {
+      const folderName = faker.word.words(1)
+      const documentContentPage = new DocumentContentPage(page)
+      await documentContentPage.clickAddFolderButton()
+      await documentContentPage.createDocumentSpaceMembersToJustMember(folderName)
+      await documentContentPage.checkIfUserCanSelectSpace(folderName, false)
```

Comment:
There are multiple ways to add new documents to a space. New document button (wizard) is only one of them. Other ways are by adding a new document of the space or child document of a document in the space from the context menu of the space/document respectively. To check the requirement we should check that all of these options are not available.